collect_app/src/main/java/org/odk/collect/android/tasks/SaveToDiskTask.java

```diff
@@ -129,8 +129,10 @@ protected SaveResult doInBackground(Void... nothing) {
 
             // attempt to remove any scratch file
             File shadowInstance = savepointFile(formController.getInstancePath());
+            File shadowFormIndex = savepointFormIndexIndexFile(formController.getInstancePath());
             if (shadowInstance.exists()) {
                 FileUtils.deleteAndReport(shadowInstance);
+                FileUtils.deleteAndReport(shadowFormIndex);
```

Comment:
Just because the instance savepoint exists doesn't mean this file will.

Reply:
Yes we shouldn't check for existing here at all since it's checked here: https://github.com/opendatakit/collect/blob/master/collect_app/src/main/java/org/odk/collect/android/utilities/FileUtils.java#L398

Reply:
Even better.